src/guides/v2.3/frontend-dev-guide/themes/admin_theme_create.md

```diff
@@ -83,6 +83,8 @@ To customize the Admin theme logo:
            <referenceBlock name="logo">
                <arguments>
                    <argument name="logo_image_src" xsi:type="string">images/custom-logo.svg</argument>
+                   <argument name="logo_img_width" xsi:type="number">150</argument> <!-- Add custom logo width -->
```

Comment:
Hi @abrarpathan19. Thank you for your contribution. Could you, please, refer to the functional part of the codebase that operates with `logo_img_width` and `logo_img_height` parameters? I'm unable to find this logic in the `Magento\Backend\Block\Page\Header` class or related functional parts. 

Thank you!

Reply:
Hello @rogyar,

Thanks for review. This is the customize the Admin theme logo section so, I have added custom with and height for the logo.

Reply:
Be aware that `logo_img_width` and `logo_img_height` are being replaced with `logo_width` and `logo_height` as of Magento 2.3.5. See https://github.com/magento/magento2/issues/25042#issuecomment-616625004 for more info.

Reply:
Good eye, @hostep. @abrarpathan19, if you want to get this change into 2.3.5-develop, that would be great.

Reply:
Hi @dobooth. Since the mentioned parameters were removed from the codebase and replaced by new parameters (`logo_width` and `logo_height`), we may leave it in the master branch but adjust the parameter names to reflect the actual state in the codebase. @abrarpathan19 would you make this change, please?